.github/pr-prefix-labeler.yml

```diff
@@ -0,0 +1,10 @@
+"[BUG]": "T.0 - Bug"
```

Comment:
Would also add `"[FIX]": "T.0 - Bug"` since I feel like this might be a prefix that people use too